integrations.yml

```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+-
+  name: "CozyCal Scheduling"
```

Comment:
This needs to be appended to the bottom of the .yml file to respect historical ordering